src/index.ts

```diff
@@ -189,8 +207,13 @@ const asyncRun = async (): Promise<void> => {
     try {
         await run();
     } catch (e) {
-        core.setFailed(e);
+        let errorMessage = "System error!";
+        if (e instanceof Error) {
+            errorMessage = e.message;
+        }
+        core.error(errorMessage);
+        core.setFailed(errorMessage);
     }
 };
 
-asyncRun();
+asyncRun().then();
```

Comment:
Wofür ist das .then()?

Reply:
Das beseitigt folgende Warnung: Promise returned from asyncRun is ignored

Reply:
Ich habe eine weitere Änderung in src/index.ts gemacht (Zeile 149 ein await hinzugefügt?, bitte anschauen und kommentieren)

Reply:
Das await passt, denke ich. Das .then() ist aber trotzdem unnötig. Dann lieber die Warnung unterdrücken.

Reply:
Ok danke Alex, ich nehme das then wieder weg.